runner/analyze.py

```diff
@@ -128,6 +79,15 @@ def _perform_initialization_analysis(
     train_set: Imageset,
     cnn_analysis: Dict[str, Dict[str, FloatArray]],
 ):
+    summary = brain.scan()
+    filepath = os.path.join(cfg.system.run_dir, "brain_summary.txt")
```

Comment:
should we maybe have some kind of Path Registry somewhere to avoid having them scattered through files?
(does not need to be addressed in this PR, but perhaps might be worth an issue)

Reply:
Maybe. One day.